R/epidist_db.R

```diff
@@ -126,8 +126,13 @@ epidist_db <- function(disease = "all",
     )
   }
 
-  # read in database
-  multi_epidist <- .read_epidist_db()
+  # read database from sysdata if available if not recreate list of <epidist>
+  tryCatch(
+    error = function(cnd) {
+      multi_epidist <- .read_epidist_db()
```

Comment:
I thought the `tryCatch()` might be unnecessary, but wanted to implement it defensively as I haven't used `sysdata` in a package before and was unsure if it could go wrong and need a back up. I'll remove the `tryCatch()`.

Reply:
Resolved in dd618c238872758bd45c7c5d30cf25db7afc3f78.